tests/functional/spec/preview.spec.js

```diff
@@ -0,0 +1,41 @@
+import IntroductionPage from "../generated_pages/introduction_hub/introduction.page";
+import IntroductionPageLinear from "../generated_pages/introduction/introduction.page";
+
+describe("Introduction preview questions", () => {
+  const introductionSchema = "test_introduction_hub.json";
+  const introductionSchemaLinear = "test_introduction.json";
+  const showButton = 'button[data-ga-category="Preview Survey"]';
+  const previewSummaryContent = "#summary-accordion-1-content";
+  const previewSectionTitle = ".ons-summary__group-title";
+  const previewQuestion = ".ons-summary__item";
+
+  it("Given I start a survey, When I view the preview page, Then all preview elements should be visible", () => {
+    browser.openQuestionnaire(introductionSchemaLinear);
+    $(IntroductionPageLinear.previewQuestions()).click();
+    expect(browser.getUrl()).to.contain("questionnaire/preview");
+    expect($(previewSectionTitle).getText()).to.equal("Main section");
+    // TODO: Add data attributes to elements below so we don't rely on tags or classes that are subject to DS changes
```

Comment:
Needs `:` on both sides for IDEs to pick up.
```suggestion
    // :TODO: Add data attributes to elements below so we don't rely on tags or classes that are subject to DS changes
```